docs/querying/operators.md

```diff
@@ -241,7 +241,7 @@ vector. `by` and `without` are only used to bucket the input vector.
 `quantile` calculates the φ-quantile, the value that ranks at number φ*N among
 the N metric values of the dimensions aggregated over. φ is provided as the
 aggregation parameter. For example, `quantile(0.5, ...)` calculates the median,
-`quantile(0.95, ...)` the 95th percentile.
+`quantile(0.95, ...)` the 95th percentile. For φ = `NaN`, `NaN` is returned. 
```

Comment:
Now that you are on it, could you also add the "For φ < 0, `-Inf` is returned. For φ > 1, `+Inf` is returned." that we already had for `histogram_quantile` above.